EventHandlers/onmemberjoin.py

```diff
@@ -30,7 +28,8 @@ async def on_member_join(self, ctx: commands.Context, member: discord.Member):
 
         # Send message to dedicated channel
         channel = self.bot.get_channel(discord.utils.get(guild.channels, name=notification_channel_name).id)
```

Comment:
I would think that `discord.utils.get(guild.channels, name=notification_channel_name)` already returns a channel object, so getting the id of that channel and then using `self.bot.get_channel` would be redundant

Reply:
I have my moments, this was not one of them 😂